### PR TITLE
SOC-2077 | SOC-1372 | Discussion - add labels to upvote and replies links

### PR DESCRIPTION
**Pull request by kvas-damian**
## Links
https://wikia-inc.atlassian.net/browse/SOC-2077
https://wikia-inc.atlassian.net/browse/SOC-1372

## Description
Fix ember i18n helper to start using plurals in translations
Discussion - add labels to upvote, replies links and share, convert share-feature to pop-over

## Reviewers
@bkoval @rafalkalinski 
//cc @Warkot - i18n helper should be fixed now ;)

### CODE REVIEW

**Comment by rafalkalinski** [comment]
LGTM

**Comment by Bloodmoon24** [comment]
LGTM

File: front/main/app/helpers/i18n.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ export default Ember.Helper.helper((params, options) => {
 		if (key === 'ns') {
 			namespace = options[key];
 		} else if (options.hasOwnProperty(key)) {
-			i18nParams[key] = String(options[key]);
+			i18nParams[key] = options[key];
```
Comment (Warkot) [inline front/main/app/helpers/i18n.js]: :+1:

File: front/main/app/templates/components/discussion-post-card-detail.hbs [hunk 2]
```diff
@@ -60,21 +60,21 @@
...
+		<div class="small-2 large-3 columns count">
 			{{#link-to 'discussion.post' postId class='replies-area'}}
 				{{svg 'comments' viewBox='0 0 24 19' class='icon replies'}}
-				{{replyCount}}
+				{{i18n 'main.replies-link-label' ns='discussion' count=replyCount}}
 			{{/link-to}}
 		</div>
 	{{/if}}
-	<div class="share small-5 large-7 columns">
+	<div class="share small-5 large-6 columns">
```
Comment (bkoval) [inline front/main/app/templates/components/discussion-post-card-detail.hbs]: Does share feature still fit in one line for both L and XL screens? If yes, cool! :)

File: front/main/app/styles/module/_discussion.scss [hunk 3]
```diff
@@ -150,6 +150,7 @@ body.discussions {
 			color: $discussion-count-color;
 			display: block;
 			float: left;
+			font-weight: 500;
```
Comment (bkoval) [inline front/main/app/styles/module/_discussion.scss]: I would consider adding it to style-guide :)

File: front/main/app/templates/components/discussion-post-card-detail.hbs (hunk 2 above)
Comment (kvas-damian) [inline front/main/app/templates/components/discussion-post-card-detail.hbs]: Yes, 6 columns is enought

**Comment by bkoval** [comment]
LGTM

File: front/main/app/components/share-feature.js [hunk 4]
```diff
@@ -6,7 +6,7 @@ export default Ember.Component.extend(
 	TrackClickMixin,
 	LanguagesMixin,
 	{
-		classNames: ['share-feature'],
+		classNames: ['share-feature', 'count'],
```
Comment (bkoval) [inline front/main/app/components/share-feature.js]: How about "post-actions", as we discussed offline?

File: front/main/app/styles/module/_discussion.scss [hunk 5]
```diff
@@ -566,6 +557,13 @@ body.discussions {
 		.discussion-no-replies {
 			border-bottom: 1px solid $discussion-reply-border-color;
 		}
+
+		.replies {
+			text-align: center;
+			.replies-area {
```
Comment (bkoval) [inline front/main/app/styles/module/_discussion.scss]: Lacks new line above :)

File: front/main/app/templates/components/share-feature.hbs [hunk 6]
```diff
@@ -1,3 +1,11 @@
-{{#each currentSocialNetworks as |currentSocialNetwork|}}
-	<a {{action 'share' currentSocialNetwork}} class="{{currentSocialNetwork}} icon" href="#"></a>
-{{/each}}
+<a id="share-{{uniqueIdentifier}}" class="toggle-share-area">
+	{{svg 'share' class='icon toggle-share'}} <span class="mobile-hidden">Share</span>
```
Comment (bkoval) [inline front/main/app/templates/components/share-feature.hbs]: Remember about i18n :)

File: front/main/app/templates/components/share-feature.hbs [hunk 7]
```diff
@@ -1,3 +1,11 @@
...
+</a>
+
+{{#pop-over for=(concat 'share-' uniqueIdentifier) on='click' flow='flip'}}
+	<h6>{{header}}</h6>
+
+	{{#each currentSocialNetworks as |currentSocialNetwork|}}
+		<a {{action 'share' currentSocialNetwork}} class="{{currentSocialNetwork}} icon" href="#"></a>
```
Comment (bkoval) [inline front/main/app/templates/components/share-feature.hbs]: What do you think about adding a list of anchors?

File: front/main/tests/unit/helpers/i18n-test.js [hunk 8]
```diff
@@ -51,4 +51,14 @@ module('Unit | Helper | i18n', (hooks) => {
 
 		assert.ok(tStub.calledWith('main:string', i18nParams));
 	});
+
+	test('extra i18n int params work', (assert) => {
```
Comment (bkoval) [inline front/main/tests/unit/helpers/i18n-test.js]: :+1:

**Comment by bkoval** [comment]
LGTM!

**Comment by rafalkalinski** [comment]
LGTM

**Pull request merged**